src/index.js

```diff
@@ -56,7 +56,18 @@ module.exports = {
     'sort-vars': ['error'], // https://eslint.org/docs/rules/sort-vars
 
     // Unicorn
+    'unicorn/consistent-destructuring': 'off',
+    'unicorn/consistent-function-scoping': 'off',
+    'unicorn/no-array-callback-reference': 'off',
+    'unicorn/no-array-for-each': 'off',
+    'unicorn/no-array-reduce': 'off',
+    'unicorn/no-null': 'off',
+    'unicorn/no-useless-undefined': 'off',
+    'unicorn/numeric-separators-style': 'off',
     'unicorn/prefer-module': 'off',
+    'unicorn/prefer-node-protocol': 'off',
+    'unicorn/prefer-object-from-entries': 'off',
+    'unicorn/prevent-abbreviations': 'off',
```

Comment:
I don't mind this, I think it improves readability, but I would exclude props
```suggestion
    'unicorn/prevent-abbreviations': [
      'error',
      {
        allowList: {
          Props: true,
          props: true,
        },
      },
    ],
```